examples/variational_inference/pathfinder.myst.md

```diff
@@ -26,17 +26,23 @@ Pathfinder {cite:p}`zhang2021pathfinder` is a variational inference algorithm th
 
 This algorithm is [implemented](https://github.com/blackjax-devs/blackjax/pull/194) in [BlackJAX](https://github.com/blackjax-devs/blackjax), a library of inference algorithms for [JAX](https://github.com/google/jax). Through PyMC's JAX-backend (through [pytensor](https://github.com/pytensor-devs/pytensor)) we can run BlackJAX's pathfinder on any PyMC model with some simple wrapper code.
 
-This wrapper code is implemented in [pymcx](https://github.com/pymc-devs/pymcx/). This tutorial shows how to run Pathfinder on your PyMC model.
+This wrapper code is implemented in [pymc-experimental](https://github.com/pymc-devs/pymc-experimental/). This tutorial shows how to run Pathfinder on your PyMC model.
 
-You first need to install `pymcx`:
+You first need to install `pymc-experimental`:
 
-`pip install git+https://github.com/pymc-devs/pymcx`
+`pip install git+https://github.com/pymc-devs/pymc-experimental`
+
+Instructions for installing other packages:  
+- [jax](https://github.com/google/jax#installation)
+- [blackjax](https://pypi.org/project/blackjax/)
 
 ```{code-cell} ipython3
 import arviz as az
 import numpy as np
 import pymc as pm
-import pymcx as pmx
+import pymc_experimental as pmx
+import pytensor
```

Comment:
Does the code actually use pytensor/pt? I.e. are these imports required?

Reply:
removed unnecessary pytensor imports